### PR TITLE
dao layer implement

### PR DESCRIPTION
**Pull request by Naveenkumar-ramesh**

### CODE REVIEW

File: com/ideas2it/employee/dao/EmployeeDao.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    
+    /**
+     * Saves the employee details in above database
+     * and return true if the process is successful.
+     * 
+     * @param employee from controller
+     * @return Return the boolean value.
+     */
+    @Override
+    boolean addEmployee(Employee employee);
```
Comment (ponvel-i2i) [inline com/ideas2it/employee/dao/EmployeeDao.java:24]: Why do you add @Override annotation here?

File: com/ideas2it/employee/mapper/ModelMapper.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+import com.ideas2it.employee.model.Employee;
+import com.ideas2it.employee.model.EmployeeDTO;
+
+/**
+ * Converts DTO Employee details to model Employee details and 
+ * model Employee details to DTO Employee details.
+ * 
+ * @version 1.8 15-09-2022
+ * @author Naveenkumar R
+ */
```
Comment (ponvel-i2i) [inline com/ideas2it/employee/mapper/ModelMapper.java:14]: Its not a model mapper. Its a EmployeeMapper

File: com/ideas2it/employee/service/EmployeeManagementService.java [hunk 3]
```diff
@@ -1,25 +1,85 @@
...
 
 /**
- * Interface is used to declare the abstract method to service class
+ * Transfers the values between view and controller.
  *
  * @version 1.8 13-09-2022
  * @author Naveenkumar R
  */
-public interface EmployeeManagementService {
+public class EmployeeManagementService {
```
Comment (ponvel-i2i) [inline com/ideas2it/employee/service/EmployeeManagementService.java:20]: Why do you remove service interface

File: com/ideas2it/employee/model/EmployeeDTO.java [hunk 4]
```diff
@@ -0,0 +1,117 @@
+package com.ideas2it.employee.model;
+
```
Comment (ponvel-i2i) [inline com/ideas2it/employee/model/EmployeeDTO.java:2]: DTO package should be dto

**Review by ponvel-i2i — CHANGES_REQUESTED** [review]

**Comment by Naveenkumar-ramesh** [comment]
Bro , when i moved comments from implement to interface i mistakenly moved
it too.
then when i compiled ,found the mistake and corrected it.

**Pull request closed without merging**